test_rmw_implementation/test/test_subscription.cpp

```diff
@@ -530,6 +531,12 @@ TEST_F(CLASSNAME(TestSubscriptionUse, RMW_IMPLEMENTATION), take_sequence) {
   rmw_message_sequence_t sequence = rmw_get_zero_initialized_message_sequence();
   rmw_ret_t ret = rmw_message_sequence_init(&sequence, count, &allocator);
   EXPECT_EQ(RMW_RET_OK, ret) << rmw_get_error_string().str;
+
+  auto seq = test_msgs__msg__Strings__Sequence__create(count);
```

Comment:
Shouldn't this have an accompanying `test_msgs__msg__Strings__Sequence__destroy(seq)`?

Reply:
Absolutely, great catch.  Should be fixed in 3db94ad .  Thanks for the review.